lib/experimental/class-wp-webfonts.php

```diff
@@ -399,8 +504,13 @@ private function get_webfonts_by_provider( array $font_families ) {
 
 				// Skip if the provider is not registered.
 				if ( ! isset( $providers[ $provider ] ) ) {
-					/* translators: %s is the provider name. */
-					trigger_error( sprintf( __( 'Webfont provider "%s" is not registered.', 'gutenberg' ), $provider ) );
+					trigger_error(
+						sprintf(
+							/* translators: %s is the provider name. */
+							esc_html__( 'Webfont provider "%s" is not registered.', 'gutenberg' ),
+							wp_kses_data( $provider )
```

Comment:
Just curious... Why did we choose to use `wp_kses_data` instead of something like `esc_html` here? `wp_kses_*` functions are generally slower than the simpler `esc_*` functions, and in this case I think `esc_html` would work just as well?

Reply:
Ignorance from my side :) Changing them!